tcms/static/js/testrun_actions.js

```diff
@@ -659,20 +659,11 @@ function showCommentForm() {
       commentsErr.html(error);
       return false;
     }
-    jQ.ajax({
-      url: '/caserun/comment-many/',
-      data: {'comment': comments, 'run': runs.join()},
-      dataType: 'json',
-      type: 'post',
-      success: function(res) {
-        if (res.rc == 0) {
-          reloadWindow();
-        } else {
-          commentsErr.html(res.response);
-          return false;
-        }
-      }
-    });
+
+    for (var i = 0; i < runs.length; i++) {
```

Comment:
try using `runs.forEach`, see other functions in JS files. 

and remove the backend view if not in use